### PR TITLE
Immediately begins exploiting the engine harder

### PR DESCRIPTION
**Pull request by Cyberboss**
Doing this now so people don't have an excuse to mess it up later

### CODE REVIEW

**Comment by ninjanomnom** [comment]
"Code Improvement"

image

**Comment by Buggy123** [comment]
I don't get it

**Comment by ninjanomnom** [comment]
good, stop looking at this and never think of it again

File: code/__HELPERS/unsorted.dm [hunk 1]
```diff
@@ -1496,3 +1496,11 @@ GLOBAL_DATUM_INIT(dview_mob, /mob/dview, new)
 			else
 				return "\[[url_encode(thing.tag)]\]"
 	return "\ref[input]"
+
+// Makes a call in the context of a different usr
+// Use sparingly
+/proc/PushUsr(mob/M, datum/callback/CB)
```
Comment (JJRcop) [inline code/__HELPERS/unsorted.dm]: World proc, an admin could use this to spoof another admin if I am getting my cards right.

**Comment by Anonmare** [comment]
>Use sparingly

[X] Doubt

**Comment by banthebantz** [comment]
why

**Comment by RemieRichards** [comment]
Did you actually test this?
The screenshot of IRC above actually proves nothing.

**Comment by AnturK** [comment]
>Adding shitcode on purpose.
Go to sleep Cyberboss

**Pull request closed without merging**